docker/Makefile

```diff
@@ -6,17 +6,14 @@ hadolint_available := $(shell hadolint --help > /dev/null 2>&1; echo $$?)
 hadolint_command := hadolint --ignore DL3008 --ignore DL3018 --ignore DL4000 --ignore DL4001
 hadolint_container := hadolint/hadolint:latest
 pwd := $(shell pwd)
-export BUNDLE_PATH = $(pwd)/.bundle/gems
-export BUNDLE_BIN = $(pwd)/.bundle/bin
-export GEMFILE = $(pwd)/Gemfile
```

Comment:
These were added to allow travis to function correctly with a Gemfile different from the Gemfile in the root of the repo. If you aren't going to have travis support this should be fine?

Reply:
I think I could make these default to the values you had before and be able to be overridden with environment variables. Does the pattern i implemented for conditionally setting `version` to `REF` seem reasonable to you?